src/GameSrc/input.c

```diff
@@ -1548,6 +1528,73 @@ uchar weapon_button_up = TRUE;
 // INTERNALS
 // ---------
 
+// -------------------------------------------------------------------------------------------
+// SetMotionCursorForMouseXY sets motion cursor for current mouse x,y position
+
+// This function is based view3d_mouse_input(); any changes there should be made here as well
```

Comment:
Is there a way to have `view3d_mouse_input` call a common function to do this work without requiring it in two places?

Reply:
I'll try to consolidate them into one function.